build/webpack.config.dev.js

```diff
@@ -14,6 +16,11 @@ module.exports = (env) => {
       hot: true,
       watchOptions: {
         poll: true
+      },
+      before: function (app, server, compiler) {
+        app.get('/cgi-bin/people.py', function (req, res) {
+          cgi(resolve(__dirname, '..', 'cgi-bin', 'people.py'))(req, res)
```

Comment:
what does this do? might be worth adding a comment, because it's petty opaque (to me at least)